src/common/BaseAnnotator.ts

```diff
@@ -36,8 +54,16 @@ export default class BaseAnnotator {
 
     store: store.AppStore;
 
-    constructor({ apiHost, container, file, intl, token }: Options) {
+    constructor({ apiHost, container, file, fileOptions = {}, intl, token }: Options) {
+        const activeId = getProp<FileOptions, string, string | null>(
```

Comment:
I think this can be simplified quite a bit, since we're using `as string` at the end, anyway:

```
const activeId: string = getProp(fileOptions, [file.id, 'annotations.activeId']);
const initialState = {
    annotations: {
        activeId
    }
}
```